webrtc-stats.html

```diff
@@ -118,9 +113,56 @@
         Basic concepts
       </h2>
       <p>
-        The stats API is defined in [[!WEBRTC]]. It is defined to return a set of objects that are
-        a subclass of the RTCStats dictionary. This is normatively defined in [[!WEBRTC]], but is
-        reproduced here for ease of reference.
+        The basic object of the stats model is the <a>stats object</a>. The following terms are
+        defined to describe it:
+      </p>
+      <dl>
+        <dt>
+          <dfn>Stats object</dfn>
+        </dt>
+        <dd>
+          <p>
+            This represents an internal object that keeps a set of data values, together with a
+            time for when these values are current. Most stats objects are related one-on-one to an
+            object defined in the WebRTC API; they may be thought of as being internal properties
+            of those objects.
+          </p>
+        </dd>
+        <dt>
+          <dfn>Stats object reference</dfn>
+        </dt>
+        <dd>
+          <p>
+            A stats object has a stable identifier "id". Other stats objects may contain references
+            to this stats object. In a <a>stats value set</a>, these references are represented by
+            a DOMString containing "id" value of the referenced stats object.
```

Comment:
Here's an example of where I think we get into trouble. Using the given definition, this sentence basically explains that (paraphrasing) one underlying object "references" a different underlying object using its id.

I find this an odd way to explain the act of referencing one RTCStats object using a reference found in a different RTCStats object. At no time does anything directly "reference" the internal object. In fact, the internal object appears redundant in the explanation.